ddtrace/propagation/http.py

```diff
@@ -41,6 +44,8 @@
 _HTTP_HEADER_B3_SAMPLED = "x-b3-sampled"
 _HTTP_HEADER_B3_FLAGS = "x-b3-flags"
 _HTTP_HEADER_TAGS = "x-datadog-tags"
+_HTTP_HEADER_TRACEPARENT = "traceparent"
```

Comment:
this should be equal to the constants values

Reply:
actually I think it shouldn't as the rest of the headers don't rely on that, and traceparent just happens to be the constant value and the exact name of the header.